crates/astria-bridge-withdrawer/tests/blackbox/helpers/test_bridge_withdrawer.rs

```diff
@@ -287,12 +266,28 @@ impl TestBridgeWithdrawerConfig {
             metrics_http_listener_addr: String::new(),
             pretty_print: true,
         };
+        if std::env::var_os("TEST_LOG").is_some() {
```

Comment:
This is bad, we cannot do that.

The reason why we had a lazylock is that the tracing subscriber sits at a global static mwmory location.

The present change can cause tests to fail/panic.